components/Subscribe/FeedDisplay.js

```diff
@@ -128,7 +130,9 @@ function FeedDisplay({ listQueryVars }) {
 
           <ListItemCopy
             icon={rssIcon}
-            textToCopy={feedUrl}
+            textToCopy={
+              feedUrl + `&source=others&utm_source=Others&utm_medium=Subcribe`
```

Comment:
What would `&utm_source=Others&utm_medium=Subcribe` do?

Reply:
It should be `&source=Others` only, I forgot to remove `&utm_source=Others&utm_medium=Subcribe` after testing.